wasm/src/bindings.cpp

```diff
@@ -321,13 +323,50 @@ class WebGLSkiaRenderer : public rive::SkiaRenderer {
                kRGBA_8888_SkColorType, nullptr, nullptr)
         .release();
   }
+
+  void clipRect(float l, float t, float r, float b) {
+    auto rect = rive::makeRenderPath();
```

Comment:
I think we might need to delete this at some point. The ones Rive makes get deleted when the artboard is destroyed.

Reply:
Good catch! Done.